dstep/translator/MacroDefinition.d

```diff
@@ -2053,6 +2053,10 @@ bool translateFunctAlias(
     {
         Identifier ident = cast(Identifier) expr.expr;
 
+        import dstep.core.Traits : isKeyword;
+        if (ident.spelling.isKeyword())
+            return false;
```

Comment:
I guess that would end up as `alias foo = assert_`. But that might be a bit pointless.